docker/mint/docker-run.sh

```diff
@@ -4,9 +4,15 @@
 
 set -ex
 export RUST_BACKTRACE=full
+
+seed=""
+if [ -n "${CFG_SEED}" ]; then
+    seed="-s $CFG_SEED"
+fi
+
 /opt/libra/bin/config-builder faucet \
     -o /opt/libra/etc \
-    -s $CFG_SEED
+    $seed
```

Comment:
in double quotes "$seed" (https://app.codacy.com/gh/libra/libra/pullRequest?prid=4843738)